R/utils_render_latex.R

```diff
@@ -214,16 +212,8 @@ create_body_component_l <- function(data) {
 
   group_rows <- create_group_rows(n_rows, groups_rows_df, context = "latex")
 
-  if (stub_available) {
-
-    default_vars <- c("::rowname", default_vars)
-
-    body <-
-      dt_stub_df_get(data = data) %>%
-      dplyr::select(rowname) %>%
-      dplyr::mutate(rowname = process_text(rowname, context = "latex")) %>%
-      dplyr::rename(`::rowname` = rowname) %>%
-      cbind(body)
+  if (stub_available && "__GT_ROWNAME_PRIVATE__" %in% names(body)) {
+    default_vars <- c("__GT_ROWNAME_PRIVATE__", default_vars)
```

Comment:
Please test with LaTeX, where the rowname contains a LaTeX special character (that would need to be escaped by us).